cli/src/commands/test.ts

```diff
@@ -33,10 +33,16 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
   const systemProgramPath = path.resolve(__dirname, "../../");
 
   try {
-    let stdout = execSync(
-      `${commandPath} ${systemProgramPath} ${process.cwd()} ${programAddress} ${programName}.so 'yarn ts-mocha -t 2000000 tests/${projectName}.ts --exit'`
-    );
-    console.log(stdout.toString().trim());
+    executeCommand({
+      command: commandPath,
+      args: [
+        systemProgramPath,
+        process.cwd(),
+        programAddress,
+        `${programName}.so`,
+        `'yarn ts-mocha 2000000 tests/${projectName}.ts --exit'`
```

Comment:
I think the -t flag is missing before 2000000
'yarn ts-mocha -t 2000000 tests/${projectName}.ts --exit'